src/pages/EntityDetails/Model/ActionLogs/ActionLogs.tsx

```diff
@@ -135,6 +206,22 @@ export default function ActionLogs() {
         let actionName = "";
         operationData.actions.forEach((actionData, index) => {
           actionName = actionData.action.name;
+          const outputMode =
+            selectedOutput[actionData.action.tag] || "STDOUT/STDERR";
+
+          const actionFullDetails = actions.find(
+            (action) => action.action.tag === actionData.action.tag
+          );
+          if (!actionFullDetails) return;
+          const stdout = (actionFullDetails.log || [])
+            .map((m) => m.message)
+            .join("\n");
+          const stderr =
+            actionFullDetails.status === "failed"
+              ? actionFullDetails.message
+              : "";
+          const StdOut = () => <span>{stdout}</span>;
+          const StdErr = () => <span style={{ color: "red" }}>{stderr}</span>;
```

Comment:
It'd probably be good do use the Vanilla red for this. You'll need to create a new class and set `color: $color-negative;`.

Reply:
I thought that creating a new scss file just for one rule is too much, I will do that then thanks!

Reply:
Having a consistent way of styling is good :+1: